docs/command/osdctl_cluster_support.md

```diff
@@ -0,0 +1,39 @@
+## osdctl cluster support
```

Comment:
**Question:** do you know how this is generated? I would suppose by `make`, but for some newer commands, this isn't auto generated.

Reply:
It is generated with `make build` as per [osdctl#installation](https://github.com/openshift/osdctl#installation).

Reply:
Thank you, I suppose something is broken with the new commands then, as the doc is not generated for them. I will look into it later if I have time.

Reply:
My bad, I didn't notice that this is a comment for a doc file change.